### DISCUSSION TITLE
Welcome to design-system Discussions!

### DISCUSSION THREAD

**Discussion by satyamyadav** [innovaccer/design-system#926] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rahulsrivastava1995** [comment]
Hi

I am trying to install this storybook project in my local, But i am facing the below issue
<img width="538" alt="screen1" src="https://user-images.githubusercontent.com/131854322/234502466-0ba3452d-0cf2-4694-8f24-7f4d62928b5e.png">
Please help me out in this

Thanks in advance
